stores/types.go

```diff
@@ -177,7 +180,54 @@ func (hs *balance) Scan(value interface{}) error {
 	return nil
 }
 
+var SQLiteTimestampFormats = []string{
+	"2006-01-02 15:04:05.999999999-07:00",
```

Comment:
do you need more than one?

Reply:
These formats are copied from the `github.com/mattn/go-sqlite3`. I believe it's actually the second format but decided to keep these exactly as they're used in the library. We can do the conversion with the format we expect if you want, I just didn't see any harm.